joomla/joomla-cms
# [4.4] Fix for multiselect.js to not to interfere with interactive elements

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/43561 .

### Summary of Changes

Fix multiselect to not to interfere with interactive elements.


### Testing Instructions
Run `npm install`.

Edit https://github.com/joomla/joomla-cms/blob/4b690a0ba68299fa9e5041478d25dcdba6e3c6c0/administrator/components/com_content/tmpl/articles/default.php#L243

Add:
```html
<details>
  <summary>Details</summary>
  Something small enough to escape casual notice.
</details>
```

Got to Articles overview `/administrator/index.php?option=com_content&view=articles`
And click on "Details" in each row.


### Actual result BEFORE applying this Pull Request
The row become selected/unselected after each click


### Expected result AFTER applying this Pull Request
The row stay unselected.


### Link to documentations
Please select:
